timing.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+subj = "03"
+sessions = ["test","retest"]
+
+# import modules
+import re #regex module
+
+for ses in sessions:
+	# open file
+	with open("./sub-%s/ses-%s/func/sub-%s_ses-%s_task-linebisection_events.tsv" %(subj, ses, subj, ses), 'r') as f:
+	    s=f.read()
+
+	# setup dictionary
+	timing_dic = {"Correct_Task": [], 
+	        "Incorrect_Task": [], 
+	        "No_Response_Task": [],
+	        "Response_Control": [],
+	        "No_Response_Control": []}
+
+	regex = re.compile(r"([\d.]+)\s+([\d.]+)\s+([\d.]+)\s+([\w]+)", re.MULTILINE)
+
+	matches = regex.finditer(s)
+
+	for m in matches:
+		timing_dic[m.groups()[3]].append(str(float(m.groups()[0])-2.5))
```

Comment:
This works, but a better practice would be to write out the onsets as they appear in the file and then explicitly adjust the timing later (an ideal place would be to add the appropriate `timing_tool.py` line into the processing script where the first volume is removed. Imagine if you were given the fMRI data (all volumes) and the timing files you write out here without any notes-what would you assume?